src/main/java/com/google/sps/api/GmailConfiguration.java

```diff
@@ -48,10 +50,15 @@
 import javax.mail.internet.MimeMessage;
 import javax.mail.internet.MimeMultipart;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /** Create and send emails from an authorized account. */
 public class GmailConfiguration {
```

Comment:
Could you point to the URL where you got this so that in the future, anyone reading the code can get more context.

Reply:
Yes, added!